6502.js

```diff
@@ -601,6 +603,24 @@ export function Cpu6502(model, dbgr, video_, soundChip_, ddNoise_, music5000_, c
             }
         }
 
+        if (this.econet) {
```

Comment:
I might see if I can rephrase this in the switch() statement below: ideally that should be the spot where most of the work is done (else we'll end up with more and more stuff here being checked before every hw access, which seems a shame.